V2_MIGRATION_GUIDE.md

```diff
@@ -181,9 +221,45 @@ Auth0.swift now only supports the [authorization code flow with PKCE](https://au
 
 The `useUniversalLink()` method was removed as well, as Universal Links [cannot be used](https://openradar.appspot.com/51091611) for OAuth redirections without user interaction since iOS 10.
 
+`useLegacyAuthentication()` and `useLegacyAuthentication(withStyle:)` were also removed, as their underlying Apple API `SFAuthenticationSession` is [deprecated](https://developer.apple.com/documentation/safariservices/sfauthenticationsession) in favor of `ASWebAuthenticationSession`. Auth0.swift uses `ASWebAuthenticationSession` by default to perform web-based authentication.
+
+#### But without `useLegacyAuthentication()` there's an alert box
```

Comment:
I would be tempted to put this content in an FAQ and link to that

Reply:
Yes, I'll eventually move it out.